### ISSUE TITLE
nanoc show-data crashes

### ISSUE THREAD

**Issue by Ecco** [nanoc/nanoc#833]
Steps to reproduce:

```
nanoc create-site FooBar
cd FooBar
nanoc show-data
```

crashes with the following error :

```
NoMethodError: undefined method `reps' for <Nanoc::Int::Item identifier="/index.html">:Nanoc::Int::Item
```

**Comment by ddfreyne** [comment]
Ack—that’s what happens when refactoring code in the presence of other, old, untested code. I have a fix in #835.

**Comment by ddfreyne** [comment]
Fixed in #835.
